cpp/log/etcd_consistent_store-inl.h

```diff
@@ -18,7 +18,7 @@ namespace {
 // etcd path constants.
 const char kClusterConfigFile[] = "/cluster_config";
 const char kUnsequencedDir[] = "/unsequenced/";
```

Comment:
Yeah, see my reply to the other comment related to this for details, but I've changed it.